Example/Account/UserAccountView/UserAccountView.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import SwiftUI
+import Combine
+import CombineFeedback
+import CombineFeedbackUI
+
+public struct UserAccountView: View {
+
+    public typealias State = UserAccountViewModel.State
+    public typealias Event = UserAccountViewModel.Event
+
+    private let context: Context<State, Event>
+
+    private let loginViewModel: LoginViewModel
+    private let registrationViewModel: RegistrationViewModel
+
+    private let authenticationSubject: PassthroughSubject<LoggedAccount?, Never> = PassthroughSubject()
+
+    private let cancellable: AnyCancellable
+
+    public init(context: Context<State, Event>) {
+        self.context = context
+        self.loginViewModel = LoginViewModel()
+        self.registrationViewModel = RegistrationViewModel()
+
+        self.cancellable = Publishers.Merge(self.loginViewModel.state.filter { $0.dismissed }.map(\.loggedAccount),
```

Comment:
Should this be done before instantiating `UserAccountView`? Connecting the view models before even the view is created?